crates/ruff_python_formatter/src/other/f_string_element.rs

```diff
@@ -37,7 +37,6 @@ impl Format<PyFormatContext<'_>> for FormatFStringElement<'_> {
             FStringElement::Expression(expression) => {
                 FormatFStringExpressionElement::new(expression, self.context).fmt(f)
             }
-            #[allow(deprecated)]
```

Comment:
Is this intentional? You don't have to reply to this comment. I just want to double check that it is.